includes/misc-functions.php

```diff
@@ -607,5 +607,12 @@ class="dashicons gmb-tooltip-icon"></span></span>',
 				esc_html__( "Would you like the featured image displayed in the marker's infowindow?", 'google-maps-builder' )
 			);
 			break;
+		case 'render_marker_show_excerpt_tooltip':
+			return sprintf( '<label class="inline_label">%1$s</label><span class="hint--top hint--top-multiline" aria-label="%2$s"><span 
+					class="dashicons gmb-tooltip-icon"></span></span>',
+				esc_html__( 'Show Excerpts', 'google-maps-builder' ),
+				esc_html__( "Would you like to display excerpts of post content instead of full content ?", 'google-maps-builder' )
```

Comment:
1. Use single quotes.
2. Update text to:

> Would you like to display the post excerpt instead of the post content?

Reply:
@DevinWalker sure I am doing the changes and will let you know soon.